my-app/src/components/CreatedListDetails/store/reducer.ts

```diff
@@ -0,0 +1,41 @@
+import { GET_LIST_DETAILS_ERROR, GET_LIST_DETAILS_SUCCESS, GET_LIST_DETAILS_REQUESTED } from './actions';
+
+export const initialListDetailsState = {
+  details: {},
+  status: null,
+  error: null,
+};
+
+enum Status {
```

Comment:
Move this type to common feature and reuse in different reducers

Reply:
done